app/Http/Requests/AccountEntityRequest.php

```
@@ -73,6 +75,26 @@ function (string $attribute, mixed $value, Closure $fail): void {
                     'required',
                     Rule::exists('currencies', 'id')
                         ->where(fn ($query) => $query->where('user_id', $this->user()->id)),
+                    // An investment transaction's commission/tax/dividend are cast to the
+                    // account's currency (MoneyCast) while price is cast to the investment's
+                    // - changing the account's currency after it's been used would silently
+                    // mismatch every existing transaction's stored values against a currency
+                    // they were never recorded in.
+                    function (string $attribute, mixed $value, Closure $fail) use ($accountEntity): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attribute'. (undefined)

(UnusedFormalParameter)